tests/run-fail/panic/unsupported_syscall.rs

```diff
@@ -1,5 +1,5 @@
 // ignore-windows: No libc on Windows
-// ignore-macos: `syscall` is not supported on macOS
+// ignore-apple: `syscall` is not supported on macOS
```

Comment:
Why do we use a different syntax that compiletest here?
Will it at least complain when it sees `ignore-foo` and doesn't understand "foo"?

Reply:
I guess if the target actually says "apple" rather than "macos" then "ingore-apple" actually makes more sense...

Reply:
yea, the ignore/only flags are essentially checking for substrings in the target triple.